public/scripts/controllers/playController.js

```diff
@@ -0,0 +1,122 @@
+angular.module('scrambleApp').
+  controller('playController', ['$scope', '$interval', 'wordFactory', 'gameStateService',
+             function($scope, $interval, wordFactory, gameStateService) {
+    //Controls the gameplay part
+
+    $scope.letters = [];
+    $scope.userInput = [];
+    $scope.score = 0;
+
+    var inputReady = true;
+    var currentWord;
+    var numCharacters; //number of characters
+    var startTime; //marks when word was given to give score
+    var endTime = Date.now() + 60999; //set end time to 60s from current time
+    var timer;
+
+    //Set scope 'letters' to a new random word
+    var getRandomWord = function() {
+      inputReady = false;
+      wordFactory.getRandomWord(function(randomWord, error) {
+        if (error) {
```

Comment:
looks like you want to just use a guard clause..

    if (error) {
      console.log(error);
      inputReady = true
      return;
    }